### PR TITLE
[Orc8r] Swagger changes for Restricted IMEIs

### PR DESCRIPTION
**Pull request by karthiksubraveti**
Signed-off-by: Karthik Subraveti <ksubraveti@fb.com>

## Summary

Swagger changes to add restricted IMEI attribute to existing network_epc configs.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml [hunk 1]
```diff
@@ -2543,7 +2550,7 @@ definitions:
     x-nullable: false
 
   plmn_config:
-    description: Record in a gateway pool
+    description: Plmn Configuration
```
Comment (mpgermano) [inline lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml]: Nit: `PLMN configuration`

File: orc8r/cloud/go/obsidian/swagger/v1/swagger.yml [hunk 2]
```diff
@@ -10191,7 +10210,7 @@ definitions:
         x-go-custom-tag: magma_alt_name:"VpnIp"
     type: object
   plmn_config:
-    description: Record in a gateway pool
+    description: Plmn Configuration
```
Comment (mpgermano) [inline orc8r/cloud/go/obsidian/swagger/v1/swagger.yml]: Nit: `PLMN configuration`

**Review by mpgermano — APPROVED** [review]

File: lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml [hunk 3]
```diff
@@ -2559,3 +2566,17 @@ definitions:
...
         x-nullable: false
+
+  imei:
+    description: IMEI value
+    type: object
+    required:
+      - tac
+    properties:
+      tac:
+        $ref: '#/definitions/tac'
```
Comment (ulaskozat) [inline lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml]: note that this is not the same as tracking area code, this is a type allocation code.

File: orc8r/cloud/go/obsidian/swagger/v1/swagger.yml [hunk 4]
```diff
@@ -8933,6 +8933,19 @@ definitions:
...
+  imei:
+    description: IMEI value
+    properties:
+      snr:
+        example: "176148"
+        pattern: ^(\d{6})$
+        type: string
+        x-nullable: false
+      tac:
+        $ref: '#/definitions/tac'
```
Comment (ulaskozat) [inline orc8r/cloud/go/obsidian/swagger/v1/swagger.yml]: this is a new field, not the same as tracking area code. let's define it as a locally scoped one just as snr here... there is no reuse by other definitions...

**Review by ulaskozat — CHANGES_REQUESTED** [review]
see the comments.

File: lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml (hunk 3 above)
Comment (karthiksubraveti) [reply lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml]: Thanks for the catch. Fixed that.

**Review by ulaskozat — CHANGES_REQUESTED** [review]
LGTM

**Review by ulaskozat — APPROVED** [review]
LGTM

**Pull request merged**